### PR TITLE
Updating banner and fixing some internationalization issues.

### PR DESCRIPTION
**Pull request by sfshaza2**
Thanks for reviewing!

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -3,6 +3,12 @@
   {% assign fl_url = '/create' -%}
   <div class="site-banner site-banner--default" role="alert">
     {% comment %} For headings use <h3> elements. {% endcomment -%}
-    Looking for inspiration? <a href='https://www.youtube.com/watch?v=MI7Rl2umeeU'>See examples of what you can build with Flutter!</a>
+    Looking for inspiration?<br>
+    Check out the
+    <a href="https://www.youtube.com/watch?v=MI7Rl2umeeU">Flutter
+    Vignettes video</a><br>
+    created by the noted design firm,
```
Comment (kwalrath) [inline src/_includes/banner.html]: No commas around "gskinner"... If you remove the following phrase, it changes the meaning of the sentence.

Maybe delete "created" and "See what's possible in Flutter!"?

File: src/_includes/banner.html [hunk 2]
```diff
@@ -6,9 +6,7 @@
     Looking for inspiration?<br>
     Check out the
     <a href="https://www.youtube.com/watch?v=MI7Rl2umeeU">Flutter
-    Vignettes video</a><br>
-    created by the noted design firm, 
-    <a href="https://gskinner.com/">gskinner</a>, for Flutter Interact.<br>
-    See what's possible in Flutter!
+    Vignettes video</a><br> by the noted design firm 
+    <a href="https://gskinner.com/">gskinner</a> for Flutter Interact.<br>
```
Comment (kwalrath) [inline src/_includes/banner.html]: I was wrong... "created" is important if we're going to keep " for Flutter Interact". (Do we need to mention Flutter Interact?)

File: src/docs/development/accessibility-and-localization/accessibility.md [hunk 3]
```diff
@@ -9,11 +9,11 @@ including those with disabilities such as blindness or motor impairment.
 
 Flutter supports three components for accessibility support:
 
-**[Large fonts](#large-fonts)**
+**[Large fonts][]**
```
Comment (kwalrath) [inline src/docs/development/accessibility-and-localization/accessibility.md]: These links aren't working:

image

File: src/docs/development/accessibility-and-localization/internationalization.md [hunk 4]
```diff
@@ -45,8 +45,8 @@ By default, Flutter only provides US English localizations.
 To add support for other languages,
 an application must specify additional `MaterialApp` properties,
 and include a separate package called
-`flutter_localizations`.  As of April 2019,
-this package supports about 52 languages.
+`flutter_localizations`.  As of February 2020,
+this package supports 77 languages.
 If you want your app to work smoothly on iOS,
 then you have to add the package
 'flutter_cupertino_localizations' as well.
```
Comment (kwalrath) [inline src/docs/development/accessibility-and-localization/internationalization.md]: ' -> `

File: src/docs/development/accessibility-and-localization/accessibility.md [hunk 5]
```diff
@@ -118,8 +120,11 @@ see the following articles written by community members:
 
 [A deep dive into Flutter's accessibility widgets]: {{site.medium}}/flutter-community/a-deep-dive-into-flutters-accessibility-widgets-eb0ef9455bc
```
Comment (kwalrath) [inline src/docs/development/accessibility-and-localization/accessibility.md]: This link is still not working

Comment (sfshaza2) [reply src/docs/development/accessibility-and-localization/accessibility.md]: Oy vey. The failure was due to the "wrong kind" of apostrophe in the article's title. It's fixed now.

**Pull request merged**